scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -313,16 +313,14 @@ def set_config_custom_tracks(display_obj: dict, build: str):
     """Set up custom public or private tracks stored in a cloud bucket or locally. These tracks were those specified in the Scout config file.
     Respect user's preferences."""
     user_obj = store.user(email=current_user.email)
-    custom_tracks_names = user_obj.get("igv_tracks")
```

Comment:
The purpose of this was in all likelihood not to "get" for each track. But the logic is the same, and the efficiency is probably not much worse. If you feel this is easier to read its all ok with me.